guides/v2.2/cloud/project/project-upgrade.md

```diff
@@ -1,259 +1,199 @@
+      deploy: |
+        php ./vendor/bin/ece-tools deploy
+      # We run post deploy hook to clean and warm the cache. Available with ECE-Tools 2002.0.10.
+      post_deploy: |
+        php ./vendor/bin/ece-tools post-deploy
+    ```
 
-### Verify or set the ADMIN_EMAIL variable {#variable}
-The environment variable `ADMIN_EMAIL` is required for upgrading and patching. This email is used for sending password reset requests and verified during when updating {{site.data.var.ece}}. See [Set environment and project variables]({{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
+1.  Add the following environment variables to the end of the `magento.app.yaml` file.
 
-### Upgrade Fastly {#fastly}
-If you have not upgraded to the latest supported version of Fastly, you should do so. Check [Supported software versions]({{ page.baseurl }}/cloud/requirements/cloud-requirements.html#cloud-arch-software) for the latest version of Fastly we recommend. For instructions, see [Upgrade Fastly]({{ page.baseurl }}/cloud/access-acct/fastly.html#upgrade).
+    ```yaml
+    variables:
+      env:
+        CONFIG__DEFAULT__PAYPAL_ONBOARDING__MIDDLEMAN_DOMAIN: 'payment-broker.magento.com'
+        CONFIG__STORES__DEFAULT__PAYMENT__BRAINTREE__CHANNEL: 'Magento_Enterprise_Cloud_BT'
+        CONFIG__STORES__DEFAULT__PAYPAL__NOTATION_CODE: 'Magento_Enterprise_Cloud'
+    ```
 
-If you do not upgrade your Fastly module, your upgraded instance will not have critical updates
-required to use Fastly and Magento services. You might also need to update the JSON format.
-This is already included in Fastly module v1.2.33 and later. For full details, see [Fastly upgrade documentation](https://github.com/fastly/fastly-magento2/blob/00f2bf042e5f708a1c3e7f49ae4f0fe71a658a76/Documentation/Guides/MAGENTO-UPGRADES.md){:target="\_blank"}.
+1. Save the file. Do not commit or push changes to your branch yet.
 
-### Update .gitignore for new generated directory {#gitignore}
-With {{site.data.var.ece}} 2.2.X, the generated folders `var/generation` and `var/di` have been moved to a `generated/` read-only directory. You should make sure the following lines are in your .gitignore file. If not, please add them and commit the changes with your branch.
-
-    /generated/*
-    !/generated/.htaccess
-
-## Back up the database {#backup-db}
+## Back up the database
 
 {% include cloud/backup-db.md %}
 
-## Verify other changes {#verify-changes}
-Verify other changes you're going to submit to source control before you start the upgrade:
-
-1.  If you haven't done so already, change to your project root directory.
-2.  Enter the following command:
-
-        git status
-3.  If there are changes you do *not* want to submit to source control, branch or stash them now.
+## Complete the upgrade
 
-## Complete the upgrade {#upgrade}
+1.  Change to your Magento base directory and set the upgrade version.
```

Comment:
Is this the same as my Magento root directory? The "base" term gives me pause for some reason. I'm not accustomed to seeing it and if it's not root, then I'm not sure what it is..